src/js/apps/globals/nav_app.js

```diff
@@ -310,9 +287,36 @@ export default RouterApp.extend({
     }
   },
   showMainNavDroplist() {
+    const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    const workspaces = currentUser.getWorkspaces();
+
+    const workspacesMenu = new Backbone.Collection(
+      workspaces.map(workspace => {
+        return {
+          id: workspace.id,
+          onSelect() {
+            Radio.trigger('event-router', `workspace:${ workspace.get('slug') }`);
+          },
+          text: workspace.get('name'),
+          icon: {
+            type: 'far',
+            icon: 'window',
+          },
+        };
+      }),
+    );
+
+    const workspacesMenuList = {
+      collection: workspacesMenu,
+      headingText: i18n.mainNavDroplist.workspacesHeading,
+      headingClassName: 'app-nav__picklist-heading',
+    };
+
     this.showChildView('navMain', new MainNavDroplist({
-      collection: appNavMenu,
+      lists: [workspacesMenuList],
       state: this.getState(),
+      selected: workspacesMenu.find({ id: currentWorkspace.id }),
```

Comment:
`myCollection.find({ id: foo.id })` is functionally the same, but more expensive than `workspacesMenu.get(currentWorkspace.id)`

However in this case the following will always be true:
`currentWorkspace === workspacesMenu.find({ id: currentWorkspace.id })`
so `state: { selected: currentWorkspace }` is all that's necessary..

it's actually an issue that the state model is passed as `state` here as the droplist has it's own state.
so this should be:

```
state: {
  selected: currentWorkspace,
  isMinimized: this.getState('isMinimized'),
}
```
then the droplist should use `getState('isMinimized')` instead of `this.getOption('state').get('isMinimized');`